config/error_handler.go

```diff
@@ -11,7 +11,7 @@ var _ Inline = &ErrorHandler{}
 
 // ErrorHandler represents a subset of Endpoint.
 type ErrorHandler struct {
-	Kinds     []string
+	Kinds     []string  // FIXME: Why not `hcl:"type,label"`?
```

Comment:
Just ask, labels have a fixed order and MUST be provided if defined within schema. Count MUST match and cant be one and then two etc. This implementation is flexible.
```suggestion
	Kinds     []string 
```